manifests/pod.yaml

```diff
@@ -1,34 +1,121 @@
-# This pod template is used in the Jenkins pipeline by the Kubernetes plugin
+# This pod template provides the same functionality as pod.yaml, but includes a
+# a configMap to adding a CA-Certifiate to trust (i.e. private Git Repo).
```

Comment:
Did you mean to make this a different file? Changing from a pod definition to a template will break the pipeline unless we adjust the `Jenkinsfile` too, right?

Reply:
That was originally a separate file. I've fixed that.